### PR TITLE
in-formant: init at 2021-06-30

### PR DESCRIPTION
**Pull request by ckiee**
###### Notes

- Requires merge of #131526 for maintainer username change.
- Pinned to an older version because of missing Qt 6, #108008.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/in-formant/default.nix [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    mkdir -p $out/bin
+    cp in-formant $out/bin
+  '';
+
+  meta = with lib; {
+    description = "A real-time pitch and formant tracking software";
+    homepage = "https://github.com/in-formant/in-formant";
+    license = licenses.asl20;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ ckie ];
```
Comment (bobby285271) [inline pkgs/applications/audio/in-formant/default.nix:43]: Please add yourself to `maintainers/maintainer-list.nix` with a separate commit `maintainers: add ckie`.

Comment (ckiee) [reply pkgs/applications/audio/in-formant/default.nix:43]: I mentioned that in the description:
> - Requires merge of #131526 for maintainer username change.

File: pkgs/applications/audio/in-formant/default.nix [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{ stdenv, cmake, lib, fetchFromGitHub, qt5, fftw, libtorch-bin, portaudio, eigen
+, xorg, pkg-config, autoPatchelfHook, soxr
+}:
+
+stdenv.mkDerivation rec {
+  pname = "in-formant";
+  version = "2021-06-30";
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/in-formant/default.nix:7]: This should be prefixed with unstable-

Comment (ckiee) [reply pkgs/applications/audio/in-formant/default.nix:7]: Just a note for next time since it's already merged?

**Pull request merged**